src/docs/cookbook-social-logins-and-sso/setup-oidc-proxy/index.md

```diff
@@ -18,6 +18,15 @@ into an OpenID Connect ID token that can be used to validate user login
 in Flex. With this approach, FTW will serve as an identity provider
 towards Flex.
 
+Flex verifies the ID token by fetching the JSON Web Key that is hosted
+by your FTW server and using that to unsign the token. A consequence of
+this is, that the JSON Web Key needs to be publicly available. This
+means that the proxy setup will not work in localhost. To test out the
+LinkedIn login, you should e.g.
+[deploy your FTW changes to Heroku](/tutorial-branding/deploy-to-heroku/)
+so that the .well-known endpoints are publicly reachable. You can read
```

Comment:
How about "so that the `.well-known/*` endpoints are publicly available", and then drop the "You can read more about"?